src/js/index.js

```diff
@@ -0,0 +1,43 @@
+const numberCount = 0;
+const total = document.getElementById('total');
+const operatorCheck = true;
```

Comment:
값을 재할당될 수 있는 변수는 `let`으로 선언해주셔야 합니다.!
```suggestion
let numberCount = 0;
const total = document.getElementById('total');
let operatorCheck = true;
```